strapi/strapi
# [helper-plugin]: migrate useQueryParams hook to typescript

### What does it do?

Migrate `helper-plugin/hooks/useQueryParams` to typescript.
Added test case.

### How to test it?

You can check if query params are getting set properly or not. for e.g: on Marketplace plugins/providers 

### Related issue(s)/PR(s)
https://github.com/strapi/strapi/issues/17690
